components/Tabs/src/TabBarView/MDCTabBarView.h

```diff
@@ -14,6 +14,30 @@
 
 #import <UIKit/UIKit.h>
 
+@class MDCTabBarView;
+
+/**
+ Delegate protocol for MDCTabBarView. Clients may implement this protocol to receive notifications
+ of selection changes by user action in the tab bar.
+ */
+@protocol MDCTabBarViewDelegate <UIScrollViewDelegate>
+
+@optional
+
+/**
+ Called before the selected item changes by user action. Return YES to allow the selection. If not
+ implemented all items changes are allowed.
+ */
+- (BOOL)tabBarView:(nonnull MDCTabBarView *)tabBarView
+  shouldSelectItem:(nonnull UITabBarItem *)item;
+
+/**
+ Called when the selected item changes by user action.
+ */
+- (BOOL)tabBarView:(nonnull MDCTabBarView *)tabBarView didSelectItem:(nonnull UITabBarItem *)item;
```

Comment:
This should be changed to
/**
 Called when the selected item changes by user action. This method is not called for programmatic
 changes to the tab bar's selected item.
 */
- (void)tabBar:(nonnull MDCTabBar *)tabBar didSelectItem:(nonnull UITabBarItem *)item;

Reply:
Done.

Reply:
Please change the return value to void.